app/ui/core/src/commonMain/kotlin/com/moneymanager/ui/util/CurrencyFormatting.kt

```
@@ -14,21 +14,18 @@ import com.moneymanager.currency.Currency as CurrencyFormatter
 fun formatAmount(
     amount: Number,
     currency: Currency,
-): String {
-    return try {
+): String =
+    try {
         CurrencyFormatter(currency.code).format(amount)
     } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)